### PR TITLE
Release v0.38.15

### PR DESCRIPTION
**Pull request by alesforz**
[CHANGELOG](https://github.com/cometbft/cometbft/blob/release/v0.38.15/CHANGELOG.md#v03815).

Part of #4444.

### CODE REVIEW

**Review by cason — COMMENTED** [review]
Adding a link to the previous release, kind of minor.

Need to re-run unclog to have effect.

**Review by melekes — APPROVED** [review]

**Review by cason — APPROVED** [review]
+1

File: .changelog/v0.38.15/summary.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*November 6, 2024*
+
+This release supersedes [`v0.38.14`](#v03814), which mistakenly updated the Go version to
+`1.23`, introducing an unintended breaking change. It set the Go version back
```
Comment (jmalicevic) [inline .changelog/v0.38.15/summary.md:4]: o  
```suggestion
`1.23`, introducing an unintended breaking change. It sets the Go version back
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+## v0.38.15
+
+*November 6, 2024*
+
+This release supersedes [`v0.38.14`](#v03814), which mistakenly updated the Go version to
+`1.23`, introducing an unintended breaking change. It set the Go version back
```
Comment (jmalicevic) [inline CHANGELOG.md:8]:
```suggestion
`1.23`, introducing an unintended breaking change. It sets the Go version back
```
Although I guess this is generated by unclog automatically?

**Review by jmalicevic — APPROVED** [review]
Minor comments

File: CHANGELOG.md (hunk 2 above)
Comment (melekes) [reply CHANGELOG.md:8]: yes

**Pull request merged**